devops/scripts/install_drivers.sh

```diff
@@ -39,6 +40,20 @@ function get_release() {
         | jq -r '. as $raw | try .assets[].browser_download_url catch error($raw)'
 }
 
+function get_pre_release() {
+    REPO=$1
+    HASH=$2
+    URL="https://api.github.com/repos/${REPO}/actions/artifacts?name=IGC_Ubuntu22.04_llvm14_clang-${HASH}"
```

Comment:
This is specific to `igfx`. I think we need to rename the function and remove `REPO` parameter.